color_refinement.py

```diff
@@ -221,19 +227,21 @@ def get_number_automorphisms(g: Graph) -> int:
 
     graphs = L[0]
     print("Graph: ", GRAPH)
+    automorphisms = {}
     for i in range(len(graphs)):
         for j in range(len(graphs)):
             if j == i:
                 start = time.time()
                 num = get_number_automorphisms(graphs[i])
-                print('There are', num, 'automorphisms')
+                automorphisms[graphs[i].name] = num
+                print('Graph', graphs[i].name, 'has', num, 'automorphisms')
                 print('Took', time.time() - start, 'seconds\n')
             if j > i:
                 start = time.time()
                 isomorph = is_isomorphisms(graphs[i], graphs[j])
                 print(graphs[i].name, 'and', graphs[j].name, 'isomorphic?', isomorph)
                 if isomorph:
-                    coloring = initialize_coloring(graphs[i] + graphs[j])
-                    num = count_isomorphism(graphs[i], graphs[j], coloring)
-                    print('There are', num, 'isomorphisms')
-                    print('Took', time.time() - start, 'seconds\n')
+                    # coloring = initialize_coloring(graphs[i]+graphs[j])
+                    # num = count_isomorphism(graphs[i], graphs[j], coloring)
+                    print('There are', automorphisms.get(graphs[i].name), 'isomorphisms')
```

Comment:
waarom is dit uitgecomment? Als het niet meer nodig kan je het helemaal weghalen toch?